Packages/MIES/MIES_SweepFormula_PSX.ipf

```diff
@@ -346,8 +346,6 @@ static Function/WAVE PSX_DeconvoluteSweepData(WAVE sweepData, WAVE/C psxKernelFF
 		order = PSX_DECONV_FILTER_DEF_ORDER
 	endif
 
-	ASSERT(lowFrac < highFrac, "Expected a low pass filter with lowFrac < highFrac")
```

Comment:
@t-b I think this assertion is valid because the high-pass cut-off frequency should be lower than the low-pass cut-off frequency.

I think the issue was that the operation psxDeconvFilter parameters were misnamed. LowFreq should be high pass-cutoff, and highFreq should be lowpass-cutoff

Reply:
Conclusion from chat: Readd assertion, mention in help that lowFreq is the end of the passband and highFreq the start of the reject band.

Reply:
Done.